### PR TITLE
Guide mr

### PR DESCRIPTION
**Pull request by MikepdXRider**

### CODE REVIEW

File: src/services/pronouns.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import fetch from "cross-fetch";
+
+/**
+ * Function has asynchrounous features - must be treated like a promise.
+ * @param {string} pronoun 
+ * @returns Examples from the pronouns API. If example don't exist, returns a default string.
+ */
+export default async function fetchExamples(pronoun){
```
Comment (diyanamendoza) [inline src/services/pronouns.js:8]: should we wrap fetches in try catches?

**Review by diyanamendoza — APPROVED** [review]
just wondering if we should wrap those fetches in try catches?

**Review by EmilyDSarani — APPROVED** [review]
dope

**Comment by MikepdXRider** [comment]
> just wondering if we should wrap those fetches in try catches?

**Review by k-schrattenholzer — APPROVED** [review]
I'm tryin to catch how great this is!

**Pull request merged**